.github/workflows/ci.yml

```diff
@@ -20,4 +20,4 @@ jobs:
         with:
           version: v2.1
       - name: run tests
-        run: make covcheck
\ No newline at end of file
+        run: make covcheck
```

Comment:
I caused this weird UI moment when merging from main, but I don't know how

Anyway, the CI is now running